### PR TITLE
Remove registrations and some maintenance

### PR DESCRIPTION
**Pull request by sudodoki**
This PR:
- fixes some of the broken photos of coaches / moves that data to _data folder 
- removes registration form #86 (makes #60 obsolete)
- adds 'writeups' + twitter feed #85 / #66 / #49 (?)

Screenshot
image

Code is ugly, well, I'm all in for someone to clean it up. Let it be separate PR then :trollface:

### CODE REVIEW

File: public/styles/home.css [hunk 1]
```diff
@@ -720,22 +709,14 @@ footer small {
...
+
+.writeup-link {
+  font-size: 1.6em;
+}
+#ournews h2 {
+  margin-bottom: 1em;
+}
+#ournews ul {
+  margin-left: 2em
+}
```
Comment (shvaikalesh) [inline public/styles/home.css]: 🤔

Comment (sudodoki) [inline public/styles/home.css]: suggest anything else, would put it there w/o asking

**Comment by sudodoki** [comment]
@Kottans/admins_ex_owners please review

File: public/styles/home.css (hunk 1 above)
Comment (shvaikalesh) [inline public/styles/home.css]: oh sorry, everything 👌
awesome job here

File: _data/coaches.yml [hunk 2]
```diff
@@ -0,0 +1,327 @@
+placeholder:
+  - img_url: placeholder.jpg
+    name: You
+    placeholder: true
+    bio: If you think we're doing a right thing and want to contribute, be sure to drop us a line at kottans.org@gmail.com
+    # github: https://github.com/join
+    # twitter: https://twitter.com/signup
+regulars:
+  coach: &ololyay
```
Comment (zxftr45) [inline _data/coaches.yml]: I changed my logins to `zxftr45` on github and twitter

**Comment by zxftr45** [comment]
awesome 👍

File: _data/coaches.yml (hunk 2 above)
Comment (sudodoki) [inline _data/coaches.yml]: 🆗

**Pull request merged**